mito-ai/mito_ai/prompt_builders/prompt_constants.py

```diff
@@ -54,4 +59,41 @@ def cell_update_output_str(has_cell_update_output: bool) -> str:
     else:
         return ""
 
+def get_database_rules():
+    """
+    Reads the user's database configurations,
+    and returns the rules for the AI to follow.
+    """
+
+    # Get the db configuration from the user's mito folder
+
+    APP_DIR_PATH: Final[str] = os.path.join(MITO_FOLDER)
+    
+    with open(os.path.join(APP_DIR_PATH, 'db', 'connections.json'), 'r') as f:
```

Comment:
For this PoC I think this okay, but instead of hardcoding the username and password in the notebook, we should be importing from the config file. Hardcoding credentials in a notebook is obviously bad practice ...